### ISSUE TITLE
"Share quote" runs into error 404

### ISSUE THREAD

**Issue by lomaone** [friendica/friendica#9708]
Procedure:
- Any post from any network is to be shared. 
- The call is made via "Share quote"
- The share dialogue opens
- The dialogue is empty. No content is taken from the post to be shared.

Message: 
GET https://loma.ml/share/19280249?_=1609060924310 Fehler 404
in Datei: https://.../view/asset/jquery/dist/jquery.min.js?v=2020.12-rc

![Screenshot_20201227_103301_1](https://user-images.githubusercontent.com/76645483/103168095-ae031600-4830-11eb-9a4f-b0622ffb1768.png)

![Screenshot_20201227_103151_2](https://user-images.githubusercontent.com/76645483/103168098-b78c7e00-4830-11eb-9259-930278d0e5a6.png)

The problem occurs since the RC pull of 26.12.
The problem occurs with 2 separate systems

Server 1: 
Apache2; php7.3; MySQL 10.3.

Server 2:
Apache2; php7.4; MySQL 10.1.

The error could be reproduced in the browsers Firefox, Brave, Chromium and PWA (Firefox) on various computers.

**Comment by tobiasd** [comment]
Which theme are you using? In vier it seems to be working as expected.

**Comment by AlfredSK** [comment]
It is working here too. But it is unknown why. :-)

See forum (German): https://forum.friendi.ca/display/373ebf56-745f-e765-2248-87b909673868

Hmm. Maybe this is caused by an add-on?

**Comment by lomaone** [comment]
the problem occurs with "frio" and "vier"

**Comment by AlfredSK** [comment]
Concerning the add-ons: Here is a list of the add-ons you're using but I don't...

```
blackout
buffer
cookienotice
diaspora
discourse
forumdirectory
ijpost
mailstream
mastodoncustomemojis
piwik
pumpio
showmore_dyn
webrtc
xmpp
```

I would disable them to see if this issue is addon related. When the error is gone w/o these add-ons enable them again step by step until you find the wrongdoer. :-)

Btw. what even is 'mastodoncustomemojis'? There is no such add-on I know of.

Edit:
> what even is 'mastodoncustomemojis'?

Okay. Apparently, it exists in the repo but not on my node.

Edit 2:
Aha!
> Status: Unsupported

**Comment by lomaone** [comment]
Unfortunately, this does not lead to the desired result.

Mastodon Custom Emojis is an adonn by Hypolite Petovan. 
"Replace emojis shortcodes in Mastodon posts with their originating server custom emojis images."

Once it's enabled, it stays enabled. I was going to deactivate it anyway. Fits therefore

---
So, deactivated all addons. Cleared the browser cache and restarted. Reloaded the page with CTRL+F5. 

All without success.

---
Arg... the error is found.
There were problems with the relay, so I installed phpMyadmin. I have hidden the call, but with an extension that is also used by Friendica. I have now changed this, and 
it now works again.

Sorry for the inconvenience and thanks for your support
